src/components/sbb-skiplink-list/sbb-skiplink-list.tsx

```diff
@@ -1,51 +1,50 @@
-import { Component, ComponentInterface, Element, h, JSX, State, Prop } from '@stencil/core';
-import { InterfaceTitleAttributes } from '../sbb-title/sbb-title.custom';
+import { TitleLevel } from '../sbb-title';
 import {
   createNamedSlotState,
   HandlerRepository,
   namedSlotChangeHandlerAspect,
 } from '../../global/eventing';
+import { CSSResult, html, LitElement, nothing, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { spread } from '@open-wc/lit-helpers';
+import { SbbLink } from '../sbb-link';
+import Style from './sbb-skiplink-list.scss?lit&inline';
+import '../sbb-title';
 
 /**
  * @slot unnamed - Use this to provide links for the list.
```

Comment:
new notation. but you anyway need to rebase to lit-migration branch, there is already an updated string
```suggestion
 * @slot - Use this to provide links for the list.
```

Reply:
already corrected in the lit-migration branch :)